OSX/Bugsnag.xcodeproj/project.pbxproj

```diff
@@ -162,6 +162,7 @@
 		E79E6BDA1F4E3850002B35F9 /* BSG_RFC3339DateTool.m in Sources */ = {isa = PBXBuildFile; fileRef = E79E6B6E1F4E3850002B35F9 /* BSG_RFC3339DateTool.m */; };
 		E79E6BDB1F4E3850002B35F9 /* BSG_KSCrashReportFilter.h in Headers */ = {isa = PBXBuildFile; fileRef = E79E6B711F4E3850002B35F9 /* BSG_KSCrashReportFilter.h */; };
 		E79E6BDC1F4E3850002B35F9 /* BSG_KSCrashReportFilterCompletion.h in Headers */ = {isa = PBXBuildFile; fileRef = E79E6B721F4E3850002B35F9 /* BSG_KSCrashReportFilterCompletion.h */; };
+		E7AB4B9E2423E184004F015A /* BugsnagOnBreadcrumbTest.m in Sources */ = {isa = PBXBuildFile; fileRef = E7AB4B9D2423E184004F015A /* BugsnagOnBreadcrumbTest.m */; };
```

Comment:
Does this need adding to tvOS too?

Reply:
We could get away with not adding it as we typically run tests from either the iOS/OSX workspace, but it can't hurt to add it in here. After the spec work there's a ticket to investigate simplifying the project configuration - currently it's quite easy to miss adding files to one of the 3 workspaces.

Reply:
I think we should add them as a matter of course until we consolidate the projects.